fixincludes/configure

```diff
@@ -4818,6 +4818,7 @@ fi
 else
   case $host in
 	i?86-*-msdosdjgpp* | \
+	aarch64-*-mingw32* | \
```

Comment:
I would replace all places where mingw32 is used to
*-mingw32*

Reply:
@eukarpov Please, check if the current state is the way as you've desribed.

Reply:
Yes, this is how it was suggested.